sharepoint/provider/provider.py

```diff
@@ -51,48 +50,27 @@ def search(query):
         if serialized_drive_item:
             results.append(serialized_drive_item)
 
-    for page, list_item in list_items:
-        serialized_list_item = serialize_list_item(page, list_item)
-
-        if serialized_list_item is not None:
-            results.append(serialized_list_item)
-
     return results
 
 
 def collect_items(sharepoint_client, hits):
     # Gather data
     drive_items = []
-    list_items = []
     for hit in hits:
-        if hit["resource"]["@odata.type"] == MicrosoftDataType.DRIVE_ITEM.value:
+        if hit["resource"]["@odata.type"] == "#microsoft.graph.driveItem":
```

Comment:
Let's set the class constant for '#microsoft.graph.driveItem' and remove the enums and consts files as they are no longer used.

Reply:
Done